python/paddle/fluid/tests/unittests/test_layer_norm_op.py

```diff
@@ -72,9 +72,16 @@ def _reference_layer_norm_grad(x, grad_y, scale, mean, var, begin_norm_axis=1):
 
 
 class TestLayerNormdOp(unittest.TestCase):
```

Comment:
Why this test just modify in original one? not in ngraph folder?

Reply:
the original test uses cuda at default. The modification is to make the use of cuda as an option. Doing this can avoid copying code. If it is preferred, we can copy the code and modify in ngraph test.

Reply:
@tensor-tang  The UT is updated to use cudnn at default and ngraph test overwrites the setup to set use_cudnn as false per recommendation.